python/ray/serve/autoscaling_policy.py

```diff
@@ -142,136 +107,59 @@ class ReplicaQueueLengthAutoscalingPolicy(AutoscalingPolicy):
     `get_decision_num_replicas` is called once every CONTROL_LOOP_PERIOD_S
     seconds.
```

Comment:
Why do we need such assumption?

Reply:
The logic for our default autoscaling policy has not changed and I think the comment is still suitable. Basically it's assuming controller's event loop runs around 0.1 seconds and use that to delay upscaling/ downscaling. I think we can follow up to clean up some of those logics later, but for the purpose of adding the custom autoscaling functionality I rather not touch this right now.